run_all_tests_no_coverage.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 export SBT_OPTS="-XX:+CMSClassUnloadingEnabled -XX:MaxMetaspaceSize=1G"
-sbt clean compile test it:test
+sbt run-all-tests
```

Comment:
Probably delete this file.  They are almost never used by anyone

Reply:
will do